### PR TITLE
PDF Build 2.0 and included PDF.zip download link on README.md

### PR DESCRIPTION
**Pull request by Bronstrom**
PDF 2.0 changes include:
- Only running on the UUG repository, not forks
- Render all markdowns to PDF, not just marp slides
- Moving the zip command inside the loop

### CODE REVIEW

**Comment by ripleymj** [comment]
Can you update the README link to `https://github.com/jmunixusers/presentations/releases/download/presentation-latest/PDF.zip` instead of the urldefense link?

File: .github/workflows/pdf-build.yml [hunk 1]
```diff
@@ -27,18 +28,21 @@ jobs:
             ${{ runner.os }}-node-
       - name: Install marp
         run: |
-          npm install --save-dev @marp-team/marp-cli
+          npm install --save-dev @marp-team/marp-cli markdown-pdf
       - name: Build PDFs
         run: |
-          for input in `grep -l 'marp: true' *.md */*.md`
+          for input in *.md */*.md;
           do
```
Comment (kylelaker) [inline .github/workflows/pdf-build.yml:35]:
```suggestion
          for input in *.md */*.md; do
```

or

```suggestion
          for input in *.md */*.md
          do
```

Please do something similar throughout. If you're putting the `do`/`then` on a new line, then you don't need the semicolon. More typical style is to put `do`/`then` on the same line I think (with the semicolon)

File: .github/workflows/pdf-build.yml [hunk 2]
```diff
@@ -27,18 +28,21 @@ jobs:
...
-            output=`echo -n $input | sed -e 's/.md$//g'`
-            echo "$output.pdf"
-            npx marp --allow-local-files "$input" -o "$output.pdf"
+            output=`echo -n $input | sed -e 's/\.md$/.pdf/g'`
+            echo "Input: $input, output $output";
+            if [[ $(grep -l 'marp: true' $input) ]];
+            then
+              npx marp --allow-local-files "$input" -o "$output"
+            else
+              npx markdown-pdf "$input" -o "$output"
```
Comment (kylelaker) [inline .github/workflows/pdf-build.yml:42]: Does `markdown-pdf` support similar theming and formatting to `marp` or is it totally different?

**Review by kylelaker — COMMENTED** [review]
If this only gets run on the parent repo, can we add recommendations to a `CONTRIBUTING.md` on how to test:
- that the PDF for your new/updated talk looks the way that you expect
- that changes to this file do what you expect

File: .github/workflows/pdf-build.yml (hunk 2 above)
Comment (Bronstrom) [reply .github/workflows/pdf-build.yml:42]: From what I've seen the formatting and syntax are fairly similar between the two.

**Comment by ripleymj** [comment]
markdown-pdf does appear to be slightly customizable via css:

```
Usage: markdown-pdf [options] <markdown-file-path>

Options:
  -V, --version                            output the version number
  <markdown-file-path>                     Path of the markdown file to convert
  -c, --cwd [path]                         Current working directory
  -p, --phantom-path [path]                Path to phantom binary
  -h, --runnings-path [path]               Path to runnings (header, footer)
  -s, --css-path [path]                    Path to custom CSS file
  -z, --highlight-css-path [path]          Path to custom highlight-CSS file
  -m, --remarkable-options [json-options]  Options to pass to remarkable
  -f, --paper-format [format]              "A3", "A4", "A5", "Legal", "Letter" or "Tabloid"
  -r, --paper-orientation [orientation]    "portrait" or "landscape"
  -b, --paper-border [measurement]         Supported dimension units are: "mm", "cm", "in", "px"
  -d, --render-delay [millis]              Delay before rendering the PDF
  -t, --load-timeout [millis]              Timeout before the page is rendered in case `page.onLoadFinished` isn't fired
  -o, --out [path]                         Path of where to save the PDF
  -h, --help                               output usage information
```

File: .github/workflows/pdf-build.yml [hunk 3]
```diff
@@ -27,18 +28,20 @@ jobs:
...
+          npm install --save-dev @marp-team/marp-cli markdown-pdf
       - name: Build PDFs
         run: |
-          for input in `grep -l 'marp: true' *.md */*.md`
-          do
-            output=`echo -n $input | sed -e 's/.md$//g'`
-            echo "$output.pdf"
-            npx marp --allow-local-files "$input" -o "$output.pdf"
+          for input in *.md */*.md; do
+            output=`echo -n $input | sed -e 's/\.md$/.pdf/g'`
```
Comment (kylelaker) [inline .github/workflows/pdf-build.yml:35]:
```suggestion
            output="${input%.md}.pdf"
```

Use Bash's native string trimming to remove everything after the _last_ `.md` and then add `.pdf` to result.

File: .github/workflows/pdf-build.yml [hunk 4]
```diff
@@ -27,18 +28,20 @@ jobs:
...
+            echo "Input: $input, output $output";
```
Comment (kylelaker) [inline .github/workflows/pdf-build.yml:36]:
```suggestion
            echo "Input: $input, output $output"
```

unnecessary semicolon

File: .github/workflows/pdf-build.yml [hunk 5]
```diff
@@ -27,18 +28,20 @@ jobs:
...
+            if [[ $(grep -l 'marp: true' $input) ]];
+            then
```
Comment (kylelaker) [inline .github/workflows/pdf-build.yml:38]:
```suggestion
            if [[ "$(grep -l 'marp: true' "$input")" ]]; then
```

Same as `for; do` and also ensure files with spaces in the name don't cause issues by quoting `$input`. Further, it's best practice to quote the output of scripts without a good reason to not do it

**Comment by kylelaker** [comment]
Consider running the script through https://www.shellcheck.net/ for more suggestions as well (or if you have `shellcheck` you could find a way to use that)

**Review by ripleymj — APPROVED** [review]

**Pull request merged**